pkgs/applications/networking/whoogle-search/default.nix

```
@@ -0,0 +1,76 @@
+{ lib
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "whoogle-search";
+  version = "0.7.2";
+
+  src = python3.pkgs.fetchPypi {
+    inherit pname version;
+    hash = "sha256-RNmHo0SUHrYGNFgKBncpszNmjW0imqxhpnH+LS/oEK0=";
+  };
+
+  postInstall = ''
+    mv $out/${python3.sitePackages}/app/static $out/${python3.sitePackages}/app/static_runtime
+
+    # FIXME: Is it possible to specify dataDir from service configuration here?
+    ln -s /var/lib/whoogle-search/static $out/${python3.sitePackages}/app/static
+  '';
+
+  postPatch = ''
+    # Remove the version pinning
+    sed -i 's/[><=]=[0-9.]*//' requirements.txt
+
+    # Remove dev packages and packages installed by dependencies
```

```suggestion
    # Remove dev packages
```
We shouldn't remove them. If any dependency no longer relies on them whoogle will break.

---

Hello @SuperSandro2000 ,
The whoogle should not break because all the necessary requirements from whoogle's [`setup.py`](https://github.com/benbusby/whoogle-search/blob/main/setup.cfg#L19=) file are specified in `propagatedBuildInputs`.

So, maybe we could remove/skip the installation from the `requirements.txt` file completely and keep the dependencies only in `propagatedBuildInputs` section. What do you think?

Or I can re-add all the subdependencies from `requirements.txt` and just remove the dev packages.

---

> So, maybe we could remove/skip the installation from the `requirements.txt` file completely and keep the dependencies only in `propagatedBuildInputs` section. What do you think?

Then we won't notice breakages on updates. Please just only remove the linting only dependencies.

---

Only the dev packages are now removed.